doc/pages/user/062-using_trigger_delivery_policies.md

```diff
@@ -0,0 +1,138 @@
+# Using Trigger Delivery Policies
+
+Trigger Delivery Policies allow customizing what Astarte is supposed to do in case of delivery errors on HTTP events
+and how to handle events which have not been delivered. More details on Trigger Delivery Policies can be found in the [Architecture Documentation](062-trigger_delivery_policies.html).
+
+Astarte allows you to install and delete Trigger Delivery Policies dynamically through its clients.
+A Trigger Delivery Policy linked to a [HTTP Trigger](060-using_triggers.html) by specifying its name in the Trigger definition.
+A Trigger can be linked to at most one Trigger Delivery Policy, but a single Trigger Delivery Policy may serve any number of Triggers.
+
+## Listing Trigger Delivery Policies
+
+At any time, you can list existing Trigger Delivery Policies in a Realm and fetch their details and definitions.
+
+### Listing and querying Trigger Delivery Policies using Realm Management API
+
+To list all existing Trigger Delivery Policies in a Realm:
+
+`GET api.<astarte base URL>/realmmanagement/v1/<realm name>/policies`
+
+```json
+{
+  "data": [
+    "my_trigger_delivery_policy",
+    "simple_trigger_delivery_policy",
+    "other_trigger_delivery_policy",
+    "retry_upon_all_errors_delivery_policy"
+  ]
+}
+```
+
+To get a Trigger definition:
+
+`GET api.<astarte base URL>/realmmanagement/v1/<realm name>/policies/simple_trigger_delivery_policy`
+
+```json
+{
+  "data": {
+    "name" : "simple_trigger_delivery_policy",
+    "maximum_capacity" : 100,
+    "error_handlers" : [
+        {
+            "on" : "any_error",
+            "strategy" : "discard"
+        }
+    ]
+  }
+}
+```
+
+## Installing a Trigger Delivery Policy
+
+To install a Trigger Delivery Policy, you need its JSON definition. Then you can install the Trigger Delivery Policy via Realm Management APIs.
+
+The name of the Trigger Delivery Policy must be unique within the Realm, or an error will be returned.
+
+### Installing a Trigger Delivery Policy using Realm Management APIs
+
+`POST api.<astarte base URL>/realmmanagement/v1/<realm name>/policies`
```

Comment:
No need for `api.`, in the rest of the documentation we use just base URL

Reply:
Actually we use both, see e.g. [user/050-query_device.md](https://github.com/astarte-platform/astarte/blob/b22499216e490575531e276e2f9d8432adf87882/doc/pages/user/050-query_device.md?plain=1#L91) or [user/060-using_triggers.md](https://github.com/astarte-platform/astarte/blob/b22499216e490575531e276e2f9d8432adf87882/doc/pages/user/060-using_triggers.md?plain=1#L54).
I'd say we make this uniform when revising the whole Astarte doc